app/models/volunteer_application.rb

```diff
@@ -0,0 +1,21 @@
+class VolunteerApplication < ActiveRecord::Base
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-]+(\.[a-z\d\-]+)*\.[a-z]+\z/i
```

Comment:
Instead of doing this I'd consider using a gem like https://github.com/balexand/email_validator

Email regexes can be quite complicated and I think being able to say:

```ruby
validates :email, email: true
```

is nice and keeps your model a bit cleaner.